### ISSUE TITLE
Multi-site: Unable to have empty desynced Replicator field

### ISSUE THREAD

**Issue by aerni** [statamic/cms#2363]
## Bug Description
There's no way to desync a Replicator field from the origin and delete all sets. When you desync the Replicator field and delete all sets, the field will be back to synced after reload.

## How to Reproduce
1. Have a multi-site installation
2. Create a blueprint with a Replicator and a set
3. Populate the content with some Replicator sets
4. Switch to another locale and desync the Replicator
5. Delete all Replicator sets and reload
6. The Replicator field will be back to synced

## Extra Detail
The only way to have a desynced and empty Replicator field is to add an empty key to the content file.

**Origin:**
```md
---
replicator:
  -
    color: Blue
    type: color
---
```

**Other locale:**
```md
---
replicator:
---
```

## Environment
Statamic 3.0.3 Pro
Laravel 7.27.0
PHP 7.4.9
aerni/snipcart dev-master

**Comment by jasonvarga** [comment]
Not limited to replicator. You cannot set a localized field to "nothing" (like an empty replicator, empty text field, etc) at the moment because Statamic strips out empty values. On the next load, it sees no value so it falls back.

We need to avoid stripping out the null values if you explicitly nulled them out. Fun!

**Comment by aerni** [comment]
Still an issue.

**Comment by jasonvarga** [comment]
Fixed in #3531
